src/main/java/com/redhat/red/build/koji/kerberos/KrbAuthenticator.java

```
@@ -110,39 +125,37 @@ public static String getKrb5ConfFilename()
 
                     if ( systemRoot != null )
                     {
-                        filename = systemRoot + File.separator + "krb5.ini";
-
+                        path = Paths.get( systemRoot, "krb5.ini" );
                     }
                 }
                 else if ( osName.startsWith( "SunOS" ) )
                 {
-                    filename = File.separator + "etc" + File.separator + "krb5" + File.separator + "krb5.conf";
+                    path = Paths.get( separator, "etc", "krb5", KRB5_CONF );
                 }
                 else
                 {
-                    filename = File.separator + "etc" + File.separator + "krb5.conf";
+                    path = Paths.get( separator, "etc", KRB5_CONF );
                 }
             }
         }
 
-        Path filenamePath = Paths.get( filename );
-
-        if ( !Files.exists( filenamePath, new LinkOption[] { } ) || !Files.isReadable( filenamePath ) )
+        if ( !Files.exists( path ) || !Files.isReadable( path ) )
         {
-            filename = null;
+            throw new IOException( "Must create or point to a krb5 configuration file before logging in" );
```

Throw exception instead of returning `null`.